search/search-domain/src/main/java/io/camunda/search/entities/UserTaskEntity.java

```diff
@@ -19,7 +19,7 @@ public final record UserTaskEntity(
     String creationTime,
     String completionTime,
     String assignee,
-    String state,
+    UserTaskState state,
```

Comment:
❓ Is this change really necessary? In the search domain, it could also just stay a String, right? This should only be about the REST API, where more "type safety" would be nice for users.

Reply:
It works without this change as well, I follow up what was develop in another entities as:
- ProcessInstanceEntity
- FlowNodeInstanceEntity
I have applied on UserTaskEntity to keep the consistency

Reply:
Interesting, I wasn't aware 👍 
I'll have a look at that as well. This is non-blocking 👍